book/src/wildcard-audit-entries.md

```diff
@@ -0,0 +1,60 @@
+# Wildcard Audit Entries
+
+This section defines the semantics of the various keys that may be specified in
+wildcard audit table entries.
```

Comment:
We should specify the table name somewhere, it's not really clear from the documentation.